File: Java Code/Basics/Recursion/Factorial.java
import java.util.Scanner;

public class Factorial{
  public static int fact(int n){
    if(n==0)
      return 1;
    return n*fact(n-1);
  }
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter the number : ");
    int n=sc.nextInt();
    if(n<0){
      System.out.println("Factorial of negative number is not defined.");
      return ;
    }
    System.out.println("Factorial of "+n+" = "+fact(n));
  }
}

File: Javascript Code/Maths Problems/Factorial.js
const n=parseInt(prompt("Enter the number : "));
let fac=1;
for(let i=2;i<=n;i++)
  fac*=i;
if(n<0)
  console.log("Factorial of negative number is not defined.");
else
  console.log(`Factorial of ${n} is ${fac}`);